### ISSUE TITLE
can you link mysql container

### ISSUE THREAD

**Issue by timfam** [docker-build/iRedMail#1]
This setup allows linking a mysql server container?
Thanks

**Comment by dashohoxha** [comment]
No, it does not. The installation and configuration of everything besides the basic system is handled by the script `./iRedMail.sh`. For more details and questions about it look at: http://www.iredmail.org/. It  supports 4 different backends for user accounts: OpenLDAP, MySQL, MariaDB and PostgreSQL.

However I agree that it would be better if data and configuration are stored outside the container itself, so that the container can be rebuilt without loosing anything. If you want you can try to improve it.
